src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -6517,13 +6520,24 @@ private boolean isJavadocVMInitError( String output )
     // Static methods
     // ----------------------------------------------------------------------
 
+    /**
+     * @return list of potential javadoc home urls to resolve <code>package-list</code> file

```

Comment:
urls --> URLs

should specify that this is returned in order of preference and indicate where URLs are looked up